python/src/viewer.py

```diff
@@ -223,10 +207,24 @@ def __init__(self,
             self._ax = [self._fig.add_subplot(111, **axes_kwargs)]
         else:
             self._ax = axes
+
         # activate axes only if wanted
         self._ax[0].axison = graph.getAxes()
 
-        has_labels = False
+        # set bounding box
+        bb = graph.getBoundingBox()
+        self._ax[0].set_xlim([bb.getLowerBound()[0], bb.getUpperBound()[0]])
+        self._ax[0].set_ylim([bb.getLowerBound()[1], bb.getUpperBound()[1]])
+
+        # set scale
+        if (graph.getLogScale() == ot.GraphImplementation.LOGX) or (graph.getLogScale() == ot.GraphImplementation.LOGXY):
+            self._ax[0].set_xscale('log')
+        if (graph.getLogScale() == ot.GraphImplementation.LOGY) or (graph.getLogScale() == ot.GraphImplementation.LOGXY):
+            self._ax[0].set_yscale('log')
+
+        # set title
+        self._ax[0].set_title(self.ToUnicode(graph.getTitle()))
```

Comment:
For me, title is a property for figure !

`
self._fig.suptitle(self.ToUnicode(graph.getTitle()))
`

Reply:
Ok. Note that the title is a bit higher now, not just on top of the graph frame.